### PR TITLE
PDP-1083: Improve pdp-distilbert performance

### PR DESCRIPTION
**Pull request by lleandroPI**

### CODE REVIEW

File: src/main/model.py [hunk 1]
```diff
@@ -155,11 +157,16 @@ def load_models(self):
...
-            print(f"Model loaded: $model_name")
-        return pipelines
+                try:
+                    sentence_transformer_model = SentenceTransformer(model_path, device=self.device)
+                    pipelines[model_name] = sentence_transformer_model
+                    if default_pipeline is None:
+                        default_pipeline = sentence_transformer_model
+                    print(f"Model loaded: {model_name}")
+                except:
+                    print(f"Could load mode: {model_name}. Skipping it.")
```
Comment (ricardo-leon-pureinsights) [inline src/main/model.py]: Message should say: "couldn't load model .... " can the error be printed with this message?

**Review by ricardo-leon-pureinsights — CHANGES_REQUESTED** [review]
The README is outdated. It doesn't contain the /encode enpoint. Could you add documentation please?

File: src/main/init_models.py [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 models = {
-    "questionAndAnswer": ['deepset/roberta-base-squad2',
-                          'mrm8488/distilbert-multi-finetuned-for-xqua-on-tydiqa',
-                          'distilbert-base-cased-distilled-squad']
+    "sentenceTransformer": ['sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2']
 }
 
-path = "../models"
+paths = {
+    "default": "C:\dev\git\models"
```
Comment (julean-pi) [inline src/main/init_models.py]: Is this the right default path? Looks like it is just for local testing

File: src/main/model.py (hunk 1 above)
Comment (lleandroPI) [reply src/main/model.py]: I think you added that message. Do you want to change it?

File: src/main/init_models.py (hunk 2 above)
Comment (lleandroPI) [reply src/main/init_models.py]: I will change it. Thanks

**Review by ricardo-leon-pureinsights — APPROVED** [review]

**Review by julean-pi — APPROVED** [review]

**Pull request merged**